ui/src/testing/stubs/runtime-api-stub.ts

```diff
@@ -8,18 +9,29 @@ import {
 } from 'generated/fetch';
 import {stubNotImplementedError} from 'testing/stubs/stub-utils';
 
+export const mockGceConfig: GceConfig = {
```

Comment:
nit: I'd characterize this as test, fake, or stub data rather than a mock. Generally I prefer reserving the term mock to refer to actual  testing mocks to avoid confusion.

opt: IN this case, I might just name it `defaultGceConfig` to match the other naming below